### PR TITLE
Initial Commit

### PR DESCRIPTION
**Pull request by cs-rohit-khune**
### Mantis#1021682 
- Validated that **Sankey** solution pack is added to the dependencies of the solution pack

### Mantis#1021694
- Validated that the MITRE data ingestion playbooks collection has been renamed to "MITRE ATT&CK 2.0.2 Demo Ingestion" and the collection ID is no longer part of the collection name

### CODE REVIEW

File: connectors/data.json [hunk 1]
```diff
@@ -133,7 +133,7 @@
                 {
                     "@context": "\/api\/3\/contexts\/WorkflowCollection",
                     "@type": "WorkflowCollection",
-                    "name": "MITRE ATT&CK 2.0.2 DemoIngestion(01e4e6b4-c34e-4fc1-b692-bb08591f1fe5)",
+                    "name": "MITRE ATT&CK 2.0.2 DemoIngestion",
```
Comment (cs-suyog-jinturkar) [inline connectors/data.json]: MITRE ATT&CK 2.0.2 DemoIngestion --> MITRE ATT&CK 2.0.2 Demo Ingestion

File: info.json [hunk 2]
```diff
@@ -108,11 +114,9 @@
                 "name": "02 - Use Case - SOC Experience"
             },
             {
-                "name": "MITRE ATT&CK 2.0.2 DemoIngestion(01e4e6b4-c34e-4fc1-b692-bb08591f1fe5)"
+                "name": "MITRE ATT&CK 2.0.2 DemoIngestion"
```
Comment (cs-suyog-jinturkar) [inline info.json]: MITRE ATT&CK 2.0.2 DemoIngestion --> MITRE ATT&CK 2.0.2 Demo Ingestion

File: playbooks/MITRE ATT&CK 2.0.2 DemoIngestion/collection.metadata.json [hunk 3]
```diff
@@ -1,7 +1,7 @@
 {
     "@context": "\/api\/3\/contexts\/WorkflowCollection",
     "@type": "WorkflowCollection",
-    "name": "MITRE ATT&CK 2.0.2 DemoIngestion(01e4e6b4-c34e-4fc1-b692-bb08591f1fe5)",
+    "name": "MITRE ATT&CK 2.0.2 DemoIngestion",
```
Comment (cs-suyog-jinturkar) [inline playbooks/MITRE ATT&CK 2.0.2 DemoIngestion/collection.metadata.json]: MITRE ATT&CK 2.0.2 DemoIngestion --> MITRE ATT&CK 2.0.2 Demo Ingestion

File: connectors/data.json (hunk 1 above)
Comment (cs-rohit-khune) [reply connectors/data.json]: Done

File: info.json (hunk 2 above)
Comment (cs-rohit-khune) [reply info.json]: Done

File: playbooks/MITRE ATT&CK 2.0.2 DemoIngestion/collection.metadata.json (hunk 3 above)
Comment (cs-rohit-khune) [reply playbooks/MITRE ATT&CK 2.0.2 DemoIngestion/collection.metadata.json]: Done

**Review by cs-suyog-jinturkar — APPROVED** [review]

**Pull request merged**